db/migrations/20230627114610_create_products_table.sql

```diff
@@ -0,0 +1,17 @@
+-- migrate:up
+CREATE TABLE products(
+  id INT NOT NULL AUTO_INCREMENT,
+  sub_category_id INT NOT NULL,
+  name VARCHAR(50) NOT NULL,
+  surface_type_id VARCHAR(50) NOT NULL,
```

Comment:
- 데이터 타입 변경
- FOREIGN KEY 추가
```suggestion
  surface_type_id INT NOT NULL,
```